vulture/utils.py

```diff
@@ -46,10 +47,10 @@ def condition_is_always_true(condition):
 
 
 def format_path(path):
-    if not path:
+    try:
+        return path.relative_to(os.curdir)
+    except ValueError:
         return path
```

Comment:
Let's add a short comment above `return path`: `# Path is not below the current directory.`